### PR TITLE
Update FAQs

### PR DESCRIPTION
**Pull request by rosieyohannan**
Some updates to the FAQs

I will make a separate ticket for the "workflows" section which requires multiple updates.

### CODE REVIEW

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -27,85 +27,62 @@ See the CircleCI [security policy]({{ site.baseurl }}/2.0/security/) for more de
...
+{: #why-is-my-job-queued }
 {:.no_toc}
-A **usage queue** forms when an organization lacks the containers to run a build. The number of available containers is determined by the plan chosen when setting up a project on CircleCI. If your builds are queuing often, you can add more containers by changing your plan.
-
-A **run queue** forms when CircleCI experiences high demand. Customer builds are placed in a run queue and processed as machines become available.
+A job might end up being **queued** because of a concurrency limit being imposed due to the plan you or your organisation are on. If your jobs are queuing often, you can consider [upgrading your plan](https://circleci.com/pricing/).
 
-In other words, you can reduce time spent in a **usage queue** by [purchasing more containers](#how-do-i-upgrade-my-container-plan-with-more-containers-to-prevent-queuing), but time spent in a **run queue** is unavoidable (though CircleCI aims to keep this as low as possible).
+**run queue** forms when CircleCI experiences high demand. Customer builds are placed in a run queue and processed as machines become available.  Time spent in a **run queue** is unavoidable (though CircleCI
```
Comment (rosieyohannan) [inline jekyll/_cci2/faq.md:80]:
```suggestion
```

File: jekyll/_cci2/faq.md [hunk 2]
```diff
@@ -27,85 +27,61 @@ See the CircleCI [security policy]({{ site.baseurl }}/2.0/security/) for more de
...
-CircleCI does not infer from your project and is moving toward a model of smart defaults with a configuration builder interface to assist with configuring all jobs in the `config.yml` file.
-
 ### Can I use CircleCI without creating base images?
-{: #can-i-use-circleci-20-without-creating-base-images }
+{: #can-i-use-circleci-without-creating-base-images }
 {:.no_toc}
-Yes, you can use one of ours! For now, but this image may be deprecated in a future release.
 
-See the [available machine images](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) section for the latest image to use. These images have the same content as the image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it is less than ideal for local testing.
+Yes, CircleCI provide a selection of "convenience images" for use with the Docker executor. For a full list, along with usage instructions visit the [CircleCI Developer Hub](https://circleci.com/developer/images) and the [CircleCI Images guide]({{site.baseurl}}/2.0/circleci-images/).
```
Comment (thomasr8) [inline jekyll/_cci2/faq.md:42]:
```suggestion
Yes, CircleCI provides a selection of "convenience images" for use with the Docker executor. For a full list, along with usage instructions, visit the [CircleCI Developer Hub](https://circleci.com/developer/images) and the [CircleCI Images guide]({{site.baseurl}}/2.0/circleci-images/).
```

File: jekyll/_cci2/faq.md [hunk 3]
```diff
@@ -27,85 +27,61 @@ See the CircleCI [security policy]({{ site.baseurl }}/2.0/security/) for more de
...
 
 ### Why are my builds queuing even though I'm on the Performance plan?
 {: #why-are-my-builds-queuing-even-though-im-on-performance-plan }
 {:.no_toc}
-In order to keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the [resource classes](https://circleci.com/docs/2.0/configuration-reference/#resource_class). If you are experiencing queuing on your builds, it's possible you are hitting these limits. Please [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request raises on these limits.
+In order to keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the [resource classes](https://circleci.com/docs/2.0/configuration-reference/#resource_class). If you are experiencing queuing on your jobs, it's possible you are hitting these limits. Please [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request raises on these limits.
```
Comment (thomasr8) [inline jekyll/_cci2/faq.md:84]:
```suggestion
In order to keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the [resource classes](https://circleci.com/docs/2.0/configuration-reference/#resource_class). If you are experiencing queuing on your jobs, it is possible you are hitting these limits. Please [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request raises on these limits.
```

**Review by thomasr8 — APPROVED** [review]
Just a couple of minor comments
